Sources/Model/Receipt/HyperwalletReceipt.swift

```diff
@@ -127,109 +125,204 @@ public struct HyperwalletReceipt: Decodable, Equatable {
 
     /// The transaction type.
     public enum HyperwalletReceiptType: String, Decodable {
-        // Generic fee types
+        /// Generic fee types
+        ///
+        /// The annual fee
         case annualFee = "ANNUAL_FEE"
+        /// The annual fee refund
         case annualFeeRefund = "ANNUAL_FEE_REFUND"
+        /// The customer service fee
         case customerServiceFee = "CUSTOMER_SERVICE_FEE"
+        /// The customer service fee refund
         case customerServiceFeeRefund = "CUSTOMER_SERVICE_FEE_REFUND"
+        /// The expedited shipping fee
         case expeditedShippingFee = "EXPEDITED_SHIPPING_FEE"
+        /// The generic fee refund
         case genericFeeRefund = "GENERIC_FEE_REFUND"
+        /// The monthly fee
         case monthlyFee = "MONTHLY_FEE"
+        /// The monthly fee refund
         case monthlyFeeRefund = "MONTHLY_FEE_REFUND"
+        /// The payment expiry fee
         case paymentExpiryFee = "PAYMENT_EXPIRY_FEE"
+        /// The payment fee
         case paymentFee = "PAYMENT_FEE"
+        /// The processing fee
         case processingFee = "PROCESSING_FEE"
+        /// The standard shipping fee
         case standardShippingFee = "STANDARD_SHIPPING_FEE"
+        /// The transfer fee
         case transferFee = "TRANSFER_FEE"
-        // Generic payment types
+        /// Generic payment types
+        ///
+        /// The adjustment
         case adjustment = "ADJUSTMENT"
+        /// The deposit
         case deposit = "DEPOSIT"
+        /// The foreign exchange
         case foreignExchange = "FOREIGN_EXCHANGE"
+        /// The manual adjustment
         case manualAdjustment = "MANUAL_ADJUSTMENT"
+        /// The payment expiration
         case paymentExpiration = "PAYMENT_EXPIRATION"
-        // Bank account-specific types
+        /// Bank account-specific types
+        ///
+        /// The bank account transfer fee
         case bankAccountTransferFee = "BANK_ACCOUNT_TRANSFER_FEE"
+        /// The bank account transfer return
         case bankAccountTransferReturn = "BANK_ACCOUNT_TRANSFER_RETURN"
+        /// The bank account transfer return fee
         case bankAccountTransferReturnFee = "BANK_ACCOUNT_TRANSFER_RETURN_FEE"
+        ///
         case transferToBankAccount = "TRANSFER_TO_BANK_ACCOUNT"
-        // Card-specific types
+        /// Card-specific types
+        ///
```

Comment:
No need to add this in document so just // are fine and remove the extra ///

Reply:
Removed extra /